code/02_calculate-measures.R

```diff
@@ -157,9 +157,11 @@ pds %<>%
 ### 7 - Save individual level file for checking ----
 pds %>% 
 write_file(path = get_mi_data_path("ldp_data", ext = "rds", test_output = test_output))
+0 # this zero stops script from running IF write_file is overwriting an existing file, re-run the section without this line and enter 1 in the console, when prompted, to overwrite file.
              
 pds %>% 
 write_file(path = get_mi_data_path("ldp_data", ext = "csv", test_output = test_output))
+0 # this zero stops script from running IF write_file is overwriting an existing file, re-run the section without this line and enter 1 in the console, when prompted, to overwrite file.
```

Comment:
I think we also need to add these comments to the publication code

Reply:
Will do

Reply:
I must have missed that when implementing `write_file` they should definitely be using `write_file` since the function is for `write_csv or write_rds`. Let me know if you want me to make that change on this branch?

Reply:
Any branch is fine. Just let me know when you've done it and I can copy it over.

Reply:
I'll make the change on a new branch to merge into master since it could be a bigger change. If we merge it into master as well then we can bring the other branches up to date.